### PR TITLE
chore: add github template issue feature

### PR DESCRIPTION
**Pull request by tormath1**
in order to prepare the public roadmap, we can submit this template to add a standard to the feature definition.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new_feature.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+### Abstract
+<!-- (Required) Only a few lines that should be understandable by everyone -->
+
+### Description
+
+#### Proposal
+<!-- (Required) What do you propose to add to Terracognita -->
+
+#### Implementation
+<!-- (Optional) May be filled later after discussion with the maintainers -->
```
Comment (xescugc) [inline .github/ISSUE_TEMPLATE/new_feature.md:17]: Which is the difference between `Description`  and `Proposal` and `Abstract`? and basically the first 2.

Comment (tormath1) [reply .github/ISSUE_TEMPLATE/new_feature.md:17]: `Description` is the main title actually. You don't write anything in this section.

`Abstract` is the high level definition: "be able to split the resources following the cloud services"

`Proposal` would be a more "detailed" description .

but it's certainly not clear enough if you ask aha

Comment (xescugc) [reply .github/ISSUE_TEMPLATE/new_feature.md:17]: Then I would remove the `Description`, no need for it. I would leave just the other 3 but even with that It feels it's "too much" but let's go with that for now and see.

**Comment by tormath1** [comment]
done

**Pull request merged**